ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -550,6 +550,23 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.poe.legacy_detect is arista.avd.defined(true) %}
    poe legacy detect
 {%     endif %}
+{%     if ethernet_interface.switchport is arista.avd.defined %}
+{%         if ethernet_interface.switchport.port_security.enable is arista.avd.defined(true) %}
+   switchport port-security
+{%         endif %}
+{%         if ethernet_interface.switchport.port_security.violation.shutdown is arista.avd.defined(true) %}
+   switchport port-security
```

Comment:
This looks wrong

Reply:
if we do 'switchport port-security violation shutdown` it gives `switchport port-security` in configurations

Reply:
It's probably the default value. So maybe this is a CLI that only takes the `no ....` form.

Reply:
So shall I insert `no switchport port-security` here?